src/components/AboutCard.jsx

```diff
@@ -21,26 +18,24 @@ function AboutCard(props) {
   const { img, name, description } = props;
 
   return (
-    <StylesProvider injectFirst>
-      <Card className={classes.card}>
-        <CardMedia
-          image={img}
-          title={`picture of ${name}`}
-          className={classes.media}
-        />
-        <CardContent>
-          <Typography variant="h6">{name}</Typography>
-          <Typography variant="body">{description}</Typography>
-        </CardContent>
-      </Card>
-    </StylesProvider>
+    <Card className={classes.card}>
+      <CardMedia
+        image={img}
+        title={`picture of ${name}`}
+        className={classes.media}
+      />
+      <CardContent>
+        <Typography variant="h6">{name}</Typography>
+        <Typography variant="body2">{description}</Typography>
+      </CardContent>
+    </Card>
   );
 }
 
 AboutCard.propTypes = {
   img: PropTypes.string.isRequired,
   name: PropTypes.string.isRequired,
-  description: PropTypes.string.isRequired,
+  description: PropTypes.node.isRequired,
```

Comment:
This allows more customized descriptions if we like via rendered types (such as my about description that has a link)